linalg/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ list(APPEND HDRS
   tlayout.hpp
   tmatrix.hpp
   ttensor.hpp
+  tensor.hpp
```

Comment:
Does `dual.hpp` need to be added here too?

Reply:
Probably :)